tests/test_pulse.py

```diff
@@ -81,4 +81,4 @@ def test_simple_transform(row_to_dict, simple_rdd, spark_context):
         'trackingProtection': False,
     }
 
-    assert row_to_dict(actual) == expected
+    assert row_to_dict(actual, recursive=False) == expected
```

Comment:
why is recursive false?

Reply:
This line uses a `Row` object, instead of the json representation.

https://github.com/mozilla/python_mozetl/blob/d92d98833482170cab672aa922f380e1e79ae527/tests/test_pulse.py#L76